### ISSUE TITLE
keep-runtime: avoid syscalls on /proc/self/cgroup

### ISSUE THREAD

**Issue by ueno** [enarx/enarx#560]
When running under strace, keep-runtime opens `/proc/self/cgroup` and issues `fcntl`:
```console
open("/proc/self/cgroup", O_RDONLY|O_CLOEXEC) = 8
fcntl(8, F_SETFD, FD_CLOEXEC)           = 0
```
We need to research whether it is possible to make this optional.

**Comment by ueno** [comment]
This is a call inside the `num_cpus` crate (`rayon` dependency), which could be avoided if we turn off the wasmtime cache system.

**Comment by ueno** [comment]
This should be fixed with #797, as we no longer create threads.
